README.md

```diff
@@ -1,97 +1,98 @@
-# Open WebRTC Toolkit Native SDK
```

Comment:
I guess you only added one line to this file, bug GitHub UI shows you changed the whole file (-97,+98). Please check whether you changed anything invisible? like line endings.